modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -954,7 +952,7 @@ definition {
 				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
 		}
 
-		task ("Then the action is triggered") {
+		task ("Then the action is triggered with succes") {
```

Comment:
We can just merge this task with the previous one. We also can remove the logic to assert success status of the action in this test since we're asserting the outcome of the action.

Reply:
okay, thanks. I'll take a look